src/app/core/utils/script-loader/script-loader.service.ts

```diff
@@ -22,9 +22,11 @@ export class ScriptLoaderService {
    *
    * @param url   script url, e.g. https://pptest.payengine.de/bridge/1.0/payengine.min.js
    * @param type  optionally set a type if it is not a classic Javascript file, e.g. 'module'
+   * @param integrity  optionally set a subresource integrity hash
+   * @returns Observable<ScriptType>
    */
 
-  load(url: string, type?: string): Observable<ScriptType> {
+  load(url: string, type?: string, integrity?: string): Observable<ScriptType> {
```

Comment:
change parameter 'type' into 'options. Options type should contain optional attributes 'type', 'integrity' and possibly 'crossOrigin'
the scriptloader changes should be an extra commit with marked as breaking changes inclusive a migration note and changes due to the changed load parameters